Resources/config/services.yml

```diff
@@ -128,9 +128,9 @@ services:
             - @ezpublish.api.service.content
             - @ezpublish.api.service.content_type
             - @ez_recommendation.rest.field.type_value
-            - {fieldIdentifiers: %ez_recommendation.field_identifiers%}
             - @ez_recommendation.field.relation_mapper
             - @?logger
+            - ["@=container.hasParameter('ez_recommendation.field_identifiers') ? parameter('ez_recommendation.field_identifiers') : 'null'"]
```

Comment:
I'm not sure if `null` should be stored inside quotation marks.
This will cast `null` value to string. Is this suppose to work like this?